README.md

```diff
@@ -34,24 +31,14 @@ High level overview is quite simple. Main components are represented in the diag
 
 * Injector: Source of HTTP traffic requests
 * Openshift Container Platform with 3scale installed
-* Upstream API: Also named as backend API, this is the final API service as an HTTP traffic endpoint. Optionally, for testing purposes, steps to use a fictional API named *echo-api* are also provided in this document.
+* Upstream API: Also named as backend API, this is the final API service as an HTTP traffic endpoint. Optionally, for testing purposes, [deploy Upstream API](doc/deploy-upstream-api.md).
```

Comment:
The link is wrong. It should be deployment/doc/deploy-upstream-api.md

Reply:
:+1: